container-core/src/main/java/ai/vespa/cloud/SystemInfo.java

```diff
@@ -1,5 +1,32 @@
 // Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package ai.vespa.cloud;
 
+import com.google.inject.Inject;
+import com.yahoo.cloud.config.ConfigserverConfig;
+
+/**
+ * Provides information about the system in which this container is running.
+ * This is available and can be injected when running in a cloud environment.
+ *
+ * @author bratseth
+ */
 public class SystemInfo {
+
+    private final Zone zone;
+
+    /** Do not use */
+    @SuppressWarnings("unused")
```

Comment:
Suppressing "unused" should be unnecessary when you have the Inject annotation.

Reply:
IntelliJ don't think so, but I guess it's clear enough with the annotation anyway.